lib/hammer_cli_foreman_leapp/preupgrade_reports.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module HammerCLIForemanLeapp
+  class PreupgradeReportsCommand < HammerCLIForeman::Command
+    resource :preupgrade_reports
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _('Id')
+        field :reported_at, _('Reported at'), Fields::Date
+        field :job_invocation_id, _('Job Invocation Id')
+        field :host_id, _('Host Id')
+
+        collection :preupgrade_report_entries, _('Entries'), hide_blank: true do
+          field :id, _('Id')
+          field :hostname, _('Host name')
+          field :title, _('Title')
+          field :actor, _('Actor')
+          field :audience, _('Audience')
+          field :severity, _('Severity')
+          field :tags, _('Tags'), Fields::List
+          field :summary, _('Summary'), Fields::LongText
+        end
+      end
+
+      build_options
+    end
+
+    class InfoCommand < HammerCLIForeman::InfoCommand
+      output PreupgradeReportsCommand::ListCommand.output_definition
+      build_options
```

Comment:
This is a bit misleading. `preupgrade_reports list` shows columns `id` and `job_invocation_id`, however `preupgrade_reports --id $ID` expects `$ID` to be a job invocation id